themes/kf/layouts/shortcodes/kubernetes-min-version.html

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
Should this also be a link?

Reply:
I've changed the K8s shortcode to a link, and I've also added a bit about shortcodes in the docs guide (the website README). You can see the README updates staged here: https://github.com/sarahmaddox/website/tree/shortcodes